.prettierrc.json

```diff
@@ -1,4 +1,10 @@
 {
     "singleQuote": true,
-    "trailingComma": "none"
+    "trailingComma": "none",
+    "printWidth": 120,
+    "tabWidth": 2,
+    "useTabs": false,
+    "semi": true,
+    "bracketSpacing": true,
+    "jsxBracketSameLine": false
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you please revert the change in this file? I'm not against it, however, it will also create changes in files not related to your change. Ideally we should keep the scope of your PR as small as possible.

Once this PR is merged - feel free to submit another PR and do this prettier config change.